packages/terra-progress-bar/src/terra-dev-site/doc/progress-bar/ProgressBar.1.doc.mdx

```diff
@@ -0,0 +1,55 @@
+import { Badge } from 'terra-progress-bar/package.json?dev-site-package';
+
+import ProgressBarDefault from '../example/ProgressBarDefault?dev-site-example';
+import ProgressBarSize from '../example/ProgressBarSize?dev-site-example';
+import ProgressBarColor from '../example/ProgressBarColor?dev-site-example';
+import ProgressBarTwoColors from '../example/ProgressBarTwoColors?dev-site-example';
+
+import ProgressBarPropsTable from 'terra-progress-bar/src/ProgressBar?dev-site-props-table';
+
+<Badge />
+
+# Terra Progress Bar
+
+The progress bar component provides users a way to display the progress of process or activity in a graphical manner. It can be modified in height and fill color. Its styling is set such that, the `ProgressBar` element will occupy full width of its parent element and will flex based on the width of the parent container.
+
+Note: The progress bar displays fill with respect to percentage (value between 0 and 100). If `max` isn't specified in the input to the component, then `value` corresponds to a percentage value.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-progress-bar`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import ProgressBar from 'terra-progress-bar';
+```
+
+## Component Features
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+* [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+* [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+
+[View component source code](https://github.com/cerner/terra-core/tree/master/packages/terra-progress-bar)
+
+## Examples
+<ProgressBarDefault title="Default" />
+<ProgressBarSize title="Size" />
+<ProgressBarColor title="Color" description="By passing in a colorClass, authors can manually set the bar color of any `<ProgressBar />` component."/>
+<ProgressBarTwoColors title="Foreground and Background Color" description="By passing in a colorClass, authors can manually set the bar color of any `<ProgressBar />` component."/>
```

Comment:
Can we add a section describing how these colorClass should be implemented under the Usage Section?

Reply:
Do we want to display CSS file to users for what CSS is being passed to the example?
Because other than showing what is in `attention` or other classes, the example seems pretty clear.

Cc / @mjhenkes

Reply:
Updated here https://github.com/cerner/terra-core/pull/2863/commits/141601c0240667ab29a64c696c2f1a2b70833b5c.